src/ftxui/dom/spinner.cpp

```diff
@@ -1,249 +1,249 @@
-        {L"[======]"},
-        {L"[===== ]"},
-        {L"[====  ]"},
-        {L"[===   ]"},
-        {L"[==    ]"},
-        {L"[=     ]"},
+        {"[      ]"},
+        {"[=     ]"},
+        {"[==    ]"},
+        {"[===   ]"},
+        {"[====  ]"},
+        {"[===== ]"},
+        {"[======]"},
+        {"[===== ]"},
+        {"[====  ]"},
+        {"[===   ]"},
+        {"[==    ]"},
+        {"[=     ]"},
     },
     {
-        {L"[      ]"},
-        {L"[=     ]"},
-        {L"[==    ]"},
-        {L"[===   ]"},
-        {L"[====  ]"},
-        {L"[===== ]"},
-        {L"[======]"},
-        {L"[ =====]"},
-        {L"[  ====]"},
-        {L"[   ===]"},
-        {L"[    ==]"},
-        {L"[     =]"},
+        {"[      ]"},
+        {"[=     ]"},
+        {"[==    ]"},
+        {"[===   ]"},
+        {"[====  ]"},
+        {"[===== ]"},
+        {"[======]"},
+        {"[ =====]"},
+        {"[  ====]"},
+        {"[   ===]"},
+        {"[    ==]"},
+        {"[     =]"},
     },
     {
-        {L"[==    ]"},
-        {L"[==    ]"},
-        {L"[==    ]"},
-        {L"[==    ]"},
-        {L"[==    ]"},
-        {L" [==   ]"},
-        {L"[  ==  ]"},
-        {L"[   == ]"},
-        {L"[    ==]"},
-        {L"[    ==]"},
-        {L"[    ==]"},
-        {L"[    ==]"},
-        {L"[    ==]"},
-        {L"[   ==] "},
-        {L"[  ==  ]"},
-        {L"[ ==   ]"},
+        {"[==    ]"},
+        {"[==    ]"},
+        {"[==    ]"},
+        {"[==    ]"},
+        {"[==    ]"},
+        {" [==   ]"},
+        {"[  ==  ]"},
+        {"[   == ]"},
+        {"[    ==]"},
+        {"[    ==]"},
+        {"[    ==]"},
+        {"[    ==]"},
+        {"[    ==]"},
+        {"[   ==] "},
+        {"[  ==  ]"},
+        {"[ ==   ]"},
     },
     {{
-         L" ─╮",
-         L"  │",
-         L"   ",
+         " ─╮",
+         "  │",
+         "   ",
      },
      {
-         L"  ╮",
-         L"  │",
-         L"  ╯",
+         "  ╮",
+         "  │",
+         "  ╯",
      },
      {
-         L"   ",
-         L"  │",
-         L" ─╯",
+         "   ",
+         "  │",
+         " ─╯",
      },
      {
-         L"   ",
-         L"   ",
-         L"╰─╯",
+         "   ",
+         "   ",
+         "╰─╯",
      },
      {
-         L"   ",
-         L"│  ",
-         L"╰─ ",
+         "   ",
+         "│  ",
+         "╰─ ",
      },
      {
-         L"╭  ",
-         L"│  ",
-         L"╰  ",
+         "╭  ",
+         "│  ",
+         "╰  ",
      },
      {
-         L"╭─ ",
-         L"│  ",
-         L"   ",
+         "╭─ ",
+         "│  ",
+         "   ",
      },
      {
-         L"╭─╮",
-         L"   ",
-         L"   ",
+         "╭─╮",
+         "   ",
+         "   ",
      }},
     {{
-         L"   /\\O  ",
-         L"    /\\/ ",
-         L"   /\\   ",
-         L"  /  \\  ",
-         L"LOL  LOL",
+         "   /\\O  ",
+         "    /\\/ ",
+         "   /\\   ",
+         "  /  \\  ",
+         "LOL  LO",
```

Comment:
nit: replace-all modified this string's content as well.

Reply:
Good catch!